USER_MANUAL.md

```diff
@@ -1549,58 +1729,912 @@ The following figure offers a visual guideline for the various parameters that c
+
+This parameter can also be set in the following files:
+
+- [Method Parameter file](#deployment_years--propagating-parameter)
+- [Site type File](#method_deploy_years-site-type-file)
+
+#### {method}_deploy_months (Sites file)
+
+**Description:** This is a propagating parameter that can be set at multiple levels of granularity. See[deploy_month](#deployment_months--propagating-parameter) for more details on what this parameter represents. In this file, this is a list of integers representing which months the specified method can be deployed on.
+
+This parameter can also be set in the following files:
+
+- [Method Parameter file](#deployment_months--propagating-parameter)
+- [Site type File](#method_deploy_months-site-type-file)
+
+#### {method}_spatial (Sites file)
+
+**Description:** This is a propagating parameter that can be set at multiple levels of granularity. See[spatial](#spatial-propagating-parameter) for more details on what this parameter represents. In this file, this is a single value, representing the chance in which the given method can detect emissions at the given site.
+
+This parameter can also be set in the following files:
+
+- [Method Parameter file](#spatial-propagating-parameter)
+- [Site type file](#method_spatial-site-type-file)
+- [Equipment file](#method_spatial-equipment-file)
+- [Source file](#method_spatial-source-file)
+
+#### {method}_survey_time (Sites file)
+
+**Description:** This is a propagating parameter that can be set at multiple levels of granularity. See [survey_time](#survey_time-propagating-parameter) for more details on what this parameter represents. In this file, this is a single value, representing how long in minutes it takes for the specified method to survey the given site.
+
+This parameter can also be set in the following files:
+
+- [Method Parameter file](#survey_time-propagating-parameter)
+- [Site type file](#method_survey_time-site-type-file)
+
+#### {method}_survey_cost (Sites file)
+
+**Description:** This is a propagating parameter that can be set at multiple levels of granularity. See [survey_cost](#per_site--propagating-parameter) for more details on what this parameter represents. In this file, this is a single value, representing the cost of the specified method to survey the given site.
+
+This parameter can also be set in the following files:
+
+- [Method Parameter file](#per_site--propagating-parameter)
+- [Site type file](#method_survey_cost-site-type-file)
+
+#### {method}_site_deployment (Sites file)
+
+**Description:** A True/False column that indicates whether the specified method will be deployed at the given site type. For example, if the value is set to `FALSE`, the emissions at the site will be simulated, but the method will not be deployed. This is similar to setting the[surveys per year](#surveys_per_year-propagating-parameter) to 0 for the site.
```

Comment:
Nit: 2 amendments.
1.  A True/False column that indicates whether the specified method will be deployed at the given site.
2 but the method will not be deployed at that site*